docs/modules/ROOT/pages/tmail-backend/jmap-extensions/publicAssets.adoc

```diff
@@ -0,0 +1,236 @@
+= Public Assets
+:navtitle: public assets
+
+Some mail providers, like Gmail, do not support base64 inline images reading in signatures. Allowing clients to store
+images as public assets to use on their TMail signature with a public URI for example could help showing the images
+correctly in other mail clients in this case.
+
+== Additions to the capability object
+
+Servers supporting the JMAP public assets extension need to advertise it via the
+`com:linagora:params:public:assets` capability.
+
+The value of this property in the JMAP session capabilities property is an object with the following field:
+
+....
+{
+    "publicAssetTotalSize": 20
+}
+....
+
+The `publicAssetTotalSize` object is an `UnsignedInt` representing the quota of public assets allowed for the user, in Mb.
```

Comment:
```suggestion
The `publicAssetTotalSize` object is an `UnsignedInt` representing the quota of public assets allowed for the user, in bytes.
``` 

Units are never specified in JMAP spec as it can lead to confusion (1000 vs 1024 for ex, or forgetting the unit).